### PR TITLE
DIV-6140: FL-DIV-GOR-ENG-00572-GeneralOrder-Updated

### PR DESCRIPTION
**Pull request by madjava**
# Description
https://tools.hmcts.net/jira/browse/DIV-6140

General Order template

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/divorce/documentgenerator/domain/TemplateConstants.java [hunk 1]
```diff
@@ -219,131 +221,129 @@
     public static final String D8_REASON_FOR_DIVORCE_ADULTERY_DETAILS_TRANS_LANG =
         "D8ReasonForDivorceAdulteryDetailsTransLang";
 
-    public static final String D8_REASON_FOR_DIVORCE_ADULTERY_WHEN_DETAILS  = "D8ReasonForDivorceAdulteryWhenDetails";
```
Comment (yarpo) [inline src/main/java/uk/gov/hmcts/reform/divorce/documentgenerator/domain/TemplateConstants.java:222]: #brave

**Review by yarpo — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/reform/divorce/documentgenerator/domain/TemplateConstants.java (hunk 1 above)
Comment (madjava) [reply src/main/java/uk/gov/hmcts/reform/divorce/documentgenerator/domain/TemplateConstants.java:222]: ...wait thats not bravely at all, let me have a look. :)

**Review by yarpo — APPROVED** [review]

File: build.gradle [hunk 2]
```diff
@@ -206,7 +207,9 @@ dependencies {
 
     compile group: 'org.springframework.security', name: 'spring-security-crypto', version: versions.springSecurityCrypto
 
-    compile('org.springframework.boot:spring-boot-starter-actuator')
+    implementation group: 'org.springframework.boot', name: 'spring-boot-starter-actuator', version: versions.springBoot
+    implementation group: 'org.yaml', name: 'snakeyaml', version: versions.snakeYaml
```
Comment (jvictorcf) [inline build.gradle]: Can you try and drop this?

File: build.gradle [hunk 3]
```diff
@@ -155,7 +155,8 @@ def versions = [
     springSecurityCrypto: '5.2.4.RELEASE',
     tomcatEmbedCore: '9.0.37',
     tomcatEmbedWebsocket: '9.0.37',
-    tomcatEmbedEl: '9.0.33'
+    tomcatEmbedEl: '9.0.33',
+    snakeYaml: '1.26'
```
Comment (jvictorcf) [inline build.gradle:159]: ditto

File: src/main/resources/application.yml [hunk 4]
```diff
@@ -301,6 +304,8 @@ management:
             show-details: "ALWAYS"
             cache:
                 time-to-live: 5000
+            probes:
+                enabled: false
```
Comment (jvictorcf) [inline src/main/resources/application.yml:308]: Can you please bring the COS comment on this config?
Don't forget to copy your changes to the example yaml file.

**Review by jvictorcf — COMMENTED** [review]
Note for next time. It would have been nice if unblocking the pipeline was in a separate PR from your story.

File: build.gradle (hunk 2 above)
Comment (madjava) [reply build.gradle]: Thanks, i have removed it, forgot to commit that change. Done now

File: build.gradle (hunk 3 above)
Comment (madjava) [reply build.gradle:159]: Ouchh! one more. Done :)

File: src/main/resources/application.yml (hunk 4 above)
Comment (madjava) [reply src/main/resources/application.yml:308]: Ok

**Review by jvictorcf — APPROVED** [review]

**Review by yarpo — APPROVED** [review]

**Review by Dizrupto — APPROVED** [review]

**Review by jvictorcf — APPROVED** [review]

**Comment by madjava** [comment]
General Order document added and updates to Spring version and actuator endpoints. Happy to merge.

**Pull request merged**